docs/how-to/h-external-k8s-connection.md

```diff
@@ -0,0 +1,70 @@
+# External K8s connection
+
+To make the Charmed Kafka K8s brokers reachable from outside the Kubernetes cluster, the charm application can manage several NodePort services:
+
+```console
+$ kubectl get services -n model
+
+| NAME                             | TYPE      | CLUSTER-IP     | CLIENT-IP   | PORT(S)                                                                         |
+|----------------------------------|-----------|----------------|-------------|---------------------------------------------------------------------------------|
+| kafka-k8s                        | ClusterIP | 10.152.183.77  |             | 65535/TCP                                                                       |
```

Comment:
No, this is the output from [kubectl](https://kubernetes.io/docs/reference/kubectl/), and that line there is the StatefulSet (aka Application) name. It could be 'my-application' if someone had ran `juju deploy kafka-k8s 3 my-application`

Reply:
Ah, my bad, I was confused by the proper markdown syntax of a table here.